### PR TITLE
Set Dispatch Queues as implemented

### PR DESCRIPTION
**Pull request by EmelyanenkoK**

### CODE REVIEW

**Comment by derockspace** [comment]
Merge pull request #370 Set Dispatch Queues as implemented

**Comment by watcharasima** [comment]
> _ไม่มีคำอธิบายให้ไว้_

**Review by Shoumo2580 — COMMENTED** [review]
EQC0lrj3O0af8GotieYsTXChA_wijIIVN7Sd_wkgYLwoH07q

File: README.md [hunk 1]
```diff
@@ -56,5 +57,4 @@ In particular "Status" below has the following sense:
 | [142](https://github.com/ton-blockchain/TEPs/pull/142/files)          | TBRC-20 Inscription Token Standard   | Contract Interface | 26.01.2024 | Proposed |
 | [145](https://github.com/ton-blockchain/TEPs/pull/145/files)          | Metadata "Hidden" render type        | Contract Interface | 26.01.2024 | ✅Deployed✅ |
 | [146](https://github.com/ton-blockchain/TEPs/pull/146/files)          | Semi-fungible token standard         | Contract Interface | 17.03.2024 | Proposed |
```
Comment (gumaxx8888) [inline README.md]: 😍

**Pull request merged**